dataland-frontend/tests/e2e/fixtures/frameworks/gdv/GdvGenerator.ts

```diff
@@ -0,0 +1,39 @@
+import { Generator } from "@e2e/utils/FakeFixtureUtils";
+
+interface YearlyTimeseriesData<InnerObj> {
```

Comment:
This here holds the code to generate fixtures for the new custom components, correct?

Reply:
Yes

Reply:
Thanks for confirming